src/Rule/AtLeastHandler.php

```diff
@@ -18,11 +18,11 @@ final class AtLeastHandler implements RuleHandlerInterface
 {
     use EmptyCheckTrait;
 
-    private FormatterInterface $formatter;
+    private TranslatorInterface $translator;
 
-    public function __construct(?FormatterInterface $formatter = null)
+    public function __construct(?TranslatorInterface $translator = null)
     {
-        $this->formatter = $formatter ?? new Formatter();
+        $this->translator = $translator ?? new FallbackTranslator();
```

Comment:
Let's avoid this workaround with configuring https://github.com/yiisoft/translator-formatter-simple by default

Reply:
Done in draft variant (tests fail for now, need to properly configure source).

Reply:
Looks good